styleguide/components/41-organisms/feedback-form/feedback-form.twig

```diff
@@ -0,0 +1,39 @@
+<div class="feedback-form {{ modifier }}">
+  <h3><button>{{ 'Can we improve something about this page?' }}<i class="icon-chevron-down" aria-hidden="true"></i></button></h3>
+  <div class="content" aria-hidden="true">
```

Comment:
Add tabIndex="-1", we'll want to put focus on the expanded element after the toggle.

Reply:
OK.